source/server/config/http/grpc_transcoding.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <string>
+
+#include "envoy/server/instance.h"
+
+#include "server/config/network/http_connection_manager.h"
+
+namespace Envoy {
+namespace Server {
+namespace Configuration {
+
+/**
+ * Config registration for the grpc transcoding filter. @see NamedHttpFilterConfigFactory.
+ */
+class GrpcTranscodingFilterConfig : public NamedHttpFilterConfigFactory {
+public:
+  HttpFilterFactoryCb createFilterFactory(const Json::Object&, const std::string&,
+                                          FactoryContext& context) override;
+  std::string name() override { return "grpc_transcoding"; };
```

Comment:
Need this to match the docs (doesn't right now). I think json should be in filter name: "grpc_json_transcoder" ?